src/structures/tss.rs

```diff
@@ -37,3 +38,34 @@ impl TaskStateSegment {
         }
     }
 }
+
+/// The given IO permissions bitmap is invalid.
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub enum InvalidIoMap {
```

Comment:
It would be great to have a `Display` implementation for this struct as a quick way to print an error.